recipes-tag/08-id-iss-collection/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set year = "2018" %}
 {% set cycle = "3" %}
-{% set version ="0" %}
+{% set version ="1" %}
 
 package:
   name: 08-id-iss-collection
   version: {{ year }}C{{ cycle }}.{{ version }}
 
 build:
-  number: 2
+  number: 1
```

Comment:
and this also?

Reply:
See above, the file is already built and is at our public channel: https://anaconda.org/lightsource2-tag/08-id-iss-collection/2018C3.1/download/linux-64/08-id-iss-collection-2018C3.1-1.tar.bz2.